website/content/docs/intro/index.mdx

```diff
@@ -2,44 +2,36 @@
+- [Get Started on VMs](https://learn.hashicorp.com/collections/consul/getting-started)
+- [HashiCorp Cloud Platform (HCP) Consul](https://learn.hashicorp.com/collections/consul/cloud-get-started)
 
-## Why Consul?
 
-Consul solves the challenges that organizations of all sizes encounter with microservices architectures. This ranges from operating in various distributed environments and geographical locations, to meeting the need of securing all application traffic.
-The world is rapidly changing and evolving, so is the computing networking layer.
 
-Today's network must quickly adapt and ensure communication is encrypted at all times. Consul enables organizations to embrace a [zero trust](https://www.hashicorp.com/solutions/zero-trust-security) model while scaling up.
-Consul can achieve all this while reducing the burden on both operators and developers through automation of crucial networking tasks
+## How does Consul work?
 
-![Diagram that explains why Consul](/img/intro_why_consul_diagram.svg)
+Consul provides a control plane that enables you to register, query, and secure services deployed across your network. The control plane is the part of the network infrastructure that maintains a central registry to track services and their respective IP addresses. It is a distributed system that runs on clusters of nodes, such as physical servers, cloud instances, virtual machines, or containers. 
 
-## What is Consul?
+Consul interacts with the data plane through proxies. The data plane is the part of the network infrastructure that processes data requests. Refer to [Consul Architecture](/docs/architecture) for details.
 
-Consul is a service mesh solution providing a full featured control plane
-with service discovery, configuration, and segmentation functionality. Each
-of these features can be used individually as needed, or they can be used
-together to build a full service mesh. Consul requires a data plane and
-supports both a proxy and native integration model. Consul ships with a
-simple built-in proxy so that everything works out of the box, but also
-supports 3rd party proxy integrations such as Envoy.
+![Basic Consul workflow](/img/what-is-consul-overview-diagram.png)
 
-Review the video below to learn more about Consul from HashiCorp's co-founder Armon.
+The core Consul workflow consists of the following stages:
+
+- **Register**: Teams add services to the Consul catalog. The catalog is a central registry that lets services automatically discover each other without requiring a human operator to modify application code, deploy additional load balancers, or hardcode IP addresses. Teams can manually [define and register services](/docs/discovery/services) using the CLI or the API, or you can automate the process in Kubernetes with [service sync](/docs/k8s/service-sync). Services can also include health checks so that Consul can monitor for unhealthy services.
```

Comment:
Suggestion:

**Register**: The Consul service catalog is a central registry which becomes the runtime source of truth for all services and their addresses. Teams can follow a small number of established patterns like deploying Consul agent [service definition files](/docs/discovery/service) along side the service, leveraging automation with tools like the Kubernetes [service sync](/docs/k8s/service-sync), or using the [CLI](/commands/services) or making [direct API calls](/api-docs/agent/service). Service definitions can also include health checks to that Consul not only knows where all of the services are, but it knows in near-real-time which instances of a service are healthy and unhealthy.

By following these standard patterns, which are easy to integrate into your existing deployment workflows, Consul will know almost immediately when you deploy, scale, or retire services."

`/docs/discovery/services` isn't CLI or API at all, it's dropping those Consul agent configuration chunks to define services via the local agent.

I also think that last sentence I added, about the workflow, is vitally important: we're easy to integrate, and if you do this, as soon as you make the decision to add, remove, or scale a service and trigger your deployment pipeline, everything else happens automatically.